openssl-sys/build.rs

```diff
@@ -336,8 +337,10 @@ RUST_LIBRESSL_251
 RUST_LIBRESSL_250
 #elif defined (LIBRESSL_VERSION_NUMBER) && LIBRESSL_VERSION_NUMBER < 0x20500000
 RUST_LIBRESSL_OLD
-#elif OPENSSL_VERSION_NUMBER >= 0x10101000
+#elif OPENSSL_VERSION_NUMBER >= 0x10200000
```

Comment:
This should look for 0x10102000.

Reply:
Is it not a reasonable assumption that 1.1.* will be backwards-compatible?

Reply:
Ah sure, that makes sense.